packages/valory/skills/olas_week_abci/behaviours.py

```diff
@@ -467,16 +467,6 @@ def _get_tweets(
         number_of_tweets_remaining_today = (
             self.params.max_tweet_pulls_allowed - number_of_tweets_pulled_today
         )
-        if number_of_tweets_remaining_today <= 0:
-            self.context.logger.info(
-                "Cannot retrieve tweets, max number of tweets reached for today"
-            )
-            return {
-                "tweets": None,
-                "error": ERROR_API_LIMITS,
-                "number_of_tweets_pulled_today": number_of_tweets_pulled_today,
-                "sleep_until": self.synchronized_data.sleep_until,
-            }
```

Comment:
Why is this removed?

Reply:
Anna realised we have a code duplucation and we were checking for the tweet limits twice.